mmv1/third_party/terraform/tests/resource_data_catalog_tag_test.go

```diff
@@ -47,6 +48,36 @@ func TestAccDataCatalogTag_update(t *testing.T) {
 	})
 }
 
+func TestAccDataCatalogTag_parent_update(t *testing.T) {
+	t.Parallel()
+
+	randStr := randString(t, 10)
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDataCatalogEntryDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataCatalogTag_parent_update(randStr, "entry"),
+			},
+			{
+				ResourceName:      "google_data_catalog_tag.basic_tag",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
```

Comment:
Won't this just recreate the resource now that it's `input: true`?

Reply:
Yes. But it would fail if `input: true` is not added. Just want to verify the code change is required for this use case